src/java.base/share/classes/java/time/Clock.java

```diff
@@ -428,6 +430,7 @@ protected Clock() {
      *  the Java epoch of 1970-01-01T00:00Z (UTC), not null
      * @throws DateTimeException if the instant cannot be obtained, not thrown by most implementations
      */
+    @Override
```

Comment:
Random question: is there any reason that `Clock` overrides `InstantSource.millis()`? They have identical impls, just with very minor javadoc tweaks.